app/views/widgets/home_page/_general_info.html

```diff
@@ -1,6 +1,6 @@
 <section class="section light-bg" id="generalInformation">
   <div class="maxWidth">
-    <h2 class="heading" skrollable skrollable-after" data-bottom="opacity:0;" data-center="opacity:1;">At Chapman, you'll discover there's an energy that runs through everything and everyone.</h2>
+    <h2 class="heading" skrollable skrollable-after data-bottom="opacity:0;" data-center="opacity:1;">At Chapman, you'll discover there's an energy that runs through everything and everyone.</h2>
```

Comment:
Same as last comment. `skrollable skrollable-after` are class names if I'm not mistaken.